index.html

```diff
@@ -376,6 +376,65 @@ <h3>History</h3>
     <div class="tiles-container">
         <div class="container container-double">
             <div class="row is-flex">
+                
+                <div class="col-xs-12">
+                    <hr class="separator hidden visible-xs" />
+                </div>
+                
+                <div class="col-xs-12" id="services">
+                    <h3 class="title">Supported Services</h3>
+                    <div class="row is-flex">
+                        <div class="tile service-tile" id="service-youtube">
+                            <div class="service-title">YouTube</div>
+                            <div class="service-description">
+                                YouTube is one of the most popular video and live streaming platforms, including music, news, gaming and much more.
+                            </div>
+                            <a class="service-link" href="https://youtube.com">visit website</a>
+                        </div>
+
+                        <div class="tile service-tile" id="service-peertube">
+                            <!--<div class="service-logo"><img src="/img/peertube-brand.png" alt="PeerTube" class="img-responsive"/></div>-->
+                            <div class="service-title">PeerTube</div>
+                            <div class="service-description">
+                                PeerTube is a decentralized video platform.<br>
+                                You can add any instance to NewPipe.
+                            </div>
+                            <a class="service-link" href="https://joinpeertube.org">visit website</a>
+                        </div>
+
+                        <div class="tile service-tile" id="service-soundcloud">
+                            <div class="service-title">SoundCloud</div>
+                            <div class="service-description">
+                                SoundCloud is a streaming platform for music.
+                            </div>
+                            <a class="service-link" href="https://soundcloud.com">visit website</a>
+                        </div>
+
+                        <div class="tile service-tile" id="service-bandcamp">
+                            <!--<div class="service-logo"></div>-->
+                            <div class="service-title">Bandcamp</div>
+                            <div class="service-description">
+                                Bandcamp is an artist friendly platform for sharing music.<br>
+                                It also offers downloads which are DRM free, making it user friendly as well.
+                            </div>
+                            <a class="service-link" href="https://bandcamp.com">visit website</a>
+                        </div>
+
+                        <div class="tile service-tile" id="service-mediaccc">
+                            <div class="service-title">Media CCC</div>
```

Comment:
I recently sent a PR for the extractor, explaining that we should rather use `media.ccc.de` instead of inventing custom titles.

```suggestion
                            <div class="service-title">media.ccc.de</div>
```

This also goes for IDs like `service-mediacccde`.